djcelery_email/tasks.py

```diff
@@ -44,7 +44,10 @@ def send_emails(messages, backend_kwargs=None, **kwargs):
     messages = [email_to_dict(m) for m in messages]
 
     conn = get_connection(backend=settings.CELERY_EMAIL_BACKEND, **combined_kwargs)
-    conn.open()
+    try:
+        conn.open()
+    except Exception:
+        logger.exception("Cannot reach CELERY_EMAIL_BACKEND %s", settings.CELERY_EMAIL_BACKEND)
```

Comment:
So you're saying that catching the exception here and letting it pass through will cause the error to be thrown below thus engaging the retry logic. So you don't need to add retry logic here?